operator/src/test/java/org/keycloak/operator/testsuite/utils/K8sUtils.java

```diff
@@ -84,6 +88,11 @@ public static void waitForKeycloakToBeReady(KubernetesClient client, Keycloak kc
                             .withName(kc.getMetadata().getName())
                             .get();
 
+                    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+                    mapper.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
+                    mapper = new ObjectMapper(new YAMLFactory().disable(YAMLGenerator.Feature.WRITE_DOC_START_MARKER));
+                    Log.infof("Current Keycloak status:\n%s", mapper.writeValueAsString(currentKc.getStatus()));
+
```

Comment:
Shouldn't this be inside a `if(Log.isInfoEnabled)` block to avoid building the mapper objects when logging is disabled?

Reply:
True. And looking at the log, I think it should log only when the status has changed. Giving it another try. Let's see what the cloud-native team thinks of this.